.github/workflows/build.yaml

```diff
@@ -0,0 +1,21 @@
+name: Flutter
+
+on:
+  schedule:
+    # “At 00:00 (UTC) on Sunday.”
+    - cron: '0 0 * * 0'
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: subosito/flutter-action@v2.4
+      with:
+        channel: 'stable'
```

Comment:
Does this mean that it's going to test against Flutter's `stable` channel?

I think the Git repository should be testing against the `master` channel since Google internally consumes these packages via Git + copybara and tries to stay current with Flutter's `master` branch.

Reply:
If we do get this PR landed, we can revisit it to run against master and stable - a matrix of jobs for [packages] x [stable, master].

Reply:
visibility_detector will not pass on stable after #367 - it's intentionally using new API that was created for the refactor.